iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/message_queue.hpp

```diff
@@ -111,9 +111,11 @@ class MessageQueue : public DesignPattern::Creation<MessageQueue, IpcChannelErro
 
   private:
     IpcChannelName_t m_name;
-    struct mq_attr m_attributes;
+    struct mq_attr m_attributes
```

Comment:
Hehehe funky ... but actually this is the wrong formatting.
I think clang-format thinks you define a struct here and therefore does a line break but actually you are defining a variable `m_attributes` of type `mq_attr`.

With c++ you do not need to write `struct` anymore and can solve this formatting issue by writing:
`mq_attr m_attributes{};` without the leading `struct`.